cluster-autoscaler/cloudprovider/aws/auto_scaling_groups.go

```diff
@@ -44,6 +44,16 @@ type asgCache struct {
 	explicitlyConfigured  map[AwsRef]bool
 }
 
+type launchTemplate struct {
+	launchTemplateName    string
```

Comment:
recommend just calling these field "name" and "version". We already know they are for the launch template since they are fields on the launchTemplate :)